language/types_values_variables.md

```diff
@@ -646,6 +986,42 @@ as negative value.
 
 Defaults to `p`
 
+##### Timespan to String
+
+| Format  | Timespan formats
+| ----    | ------------------
+| s       | formats according to the timespan format string '%D-%H:%M:%S' 
+| p       | programmatic representation - "Timespan(<quoted string>)" where <quoted string> is the result of using '%s' 
```

Comment:
Does %s produce a quoted String? - i.e. it is a bit unclear if "Timespan(12345)" or "Timespan('12345')" is produced for %p. I thought %s would produce an unquoted string.
(The same comment applies to Timestamp below).